src/BlockchainReader/Bitcoin.ts

```diff
@@ -12,7 +13,7 @@ const AnchorSectionsLengths = {
   Prefix: 4,
   StorageProtocol: 1,
   Version: 2,
-  IPFSHash: 46,
+  IPFSHash: 34,
```

Comment:
Can you add something like this to the PR description to help reviewers?

```js
console.log('before', Buffer.byteLength(bs58.decode('QmaKtaPqfWvss2ndhGdcLRc9uGcLvUUyuZeGcrDZWN58Zi')))
console.log('after', Buffer.byteLength(Buffer.from('QmaKtaPqfWvss2ndhGdcLRc9uGcLvUUyuZeGcrDZWN58Zi')))
```

Reply:
yes, sure